numpy/f2py/__init__.py

```diff
@@ -95,23 +105,21 @@ def compile(source,
              '-c',
              'import numpy.f2py as f2py2e;f2py2e.main()'] + args
         try:
-            output = subprocess.check_output(c)
-        except subprocess.CalledProcessError as exc:
-            status = exc.returncode
-            output = ''
+            cp = subprocess.run(c, stdout=subprocess.PIPE,
+                                   stderr=subprocess.PIPE)
```

Comment:
```suggestion
                                   stderr=subprocess.PIPE, text=True)
```

Reply:
Could also use `capture_output=True` instead of the two separate PIPE arguments

Reply:
That was what I did originally, but that argument was only added in Python 3.7.  What is the minimum Python version supported by Numpy these days?  I was under the impression it was at least 3.6.

Reply:
Are your referring to my first or second comment?

You're right we need 3.6 support, my mistake

Reply:
> Are your referring to my first or second comment?

I'm not sure what you mean. I only see one comment.

Reply:
The suggestion is in its own comment

Reply:
That's weird. I don't see another comment from you on this PR at all, even after refreshing the page.  The only other comment is [this one](https://github.com/numpy/numpy/pull/16205#issuecomment-626893453) which I believe is resolved by my last commit.

Reply:
[one](https://github.com/numpy/numpy/pull/16205#discussion_r423287770), [two](https://github.com/numpy/numpy/pull/16205#discussion_r423288915)

Reply:
```suggestion
                                   stderr=subprocess.PIPE,
                                   universal_newlines=True)
```
would eliminate the need for the byte-strings and `decode` below.

Reply:
I was replying to [this](https://github.com/numpy/numpy/pull/16205#discussion_r423288915) in [this comment](https://github.com/numpy/numpy/pull/16205#discussion_r424343235) as well.  `capture_output` was only added in 3.7.